openstack/cinder/templates/etc/_cinder-policy.json.tpl

```diff
@@ -52,11 +53,11 @@
   "volume_extension:extended_snapshot_attributes": "rule:context_is_editor",
   "volume_extension:volume_image_metadata": "rule:context_is_editor",
 
-  "volume_extension:quotas:show": "rule:context_is_viewer",
-  "volume_extension:quotas:update": "rule:context_is_admin",
-  "volume_extension:quotas:delete": "rule:context_is_admin",
-  "volume_extension:quota_classes": "rule:context_is_admin",
-  "volume_extension:quota_classes:validate_setup_for_nested_quota_use": "rule:context_is_admin",
+  "volume_extension:quotas:show": "rule:context_is_viewer or rule:context_is_quota_admin",
+  "volume_extension:quotas:update": "rule:context_is_quota_admin",
+  "volume_extension:quotas:delete": "rule:context_is_quota_admin",
+  "volume_extension:quota_classes": "rule:context_is_admin or rule:context_is_quota_admin",
+  "volume_extension:quota_classes:validate_setup_for_nested_quota_use": "rule:context_is_quota_admin",
```

Comment:
This last line won't work.

Reply:
It calls `cinder.quota_utls.validate_setup_for_nested_quota_use` which calls `cinder.quota.NestedDbQuotaDriver.validate_nested_setup` which calls `cinder.db.sqlalchemy.api.quota_allocated_update` which requires an admin context, which itself only uses `rule:context_is_admin` which would fail to validate.

Reply:
So this is not actually needed by Limes?